example/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="module">
```

Comment:
I would sometime like to try out ES module, Maybe we can support both worlds

We can maybe this kind of stuff to support both way:
```html
<script type="module" src="app.js"></script>
<script nomodule src="classic-app-bundle.js"></script>
```

using modules in browsers directly is also part of why i don't like TS

Reply:
Adding a CD deployed site of example/index.html so we can preview it in our PRs

Reply:
https://conflux.netlify.com/